locales/en/connection-tab.json

```diff
@@ -0,0 +1,29 @@
+{
+  "admin_url_description": "Connect to this Kafka instance over HTTPS to manage topics, consumer groups, and access. ",
+  "admin_url_label": "URL for Kafka Admin REST API",
+  "admin_url_popover_body": "Use the REST API URL to connect to this Kafka instance over HTTPS to manage topics, consumer groups, and access.",
+  "admin_url_popover_label": "URL for Kafka Admin REST API",
+  "admin_url_button_aria_label": "More information about the URL for Kafka Admin REST API",
+  "authentication_method": "Authentication method",
+  "bootstrap_server": "Bootstrap server",
+  "bootstrap_server_description": "Establish the initial connection between your client and Kafka instance.",
+  "create_service_account_to_generate_credentials": "Create a service account to generate credentials. Manage the service accounts you create on the <value>Service Accounts</value> page.",
+  "current_instance": "Accounts do not have access to this instance unless granted under the instance’s <value>Access tab</value>. The owner, the organization administrator, or users with the Allow Alter permission on this instance can manage access.",
+  "drawer_resource_tab_body_description_1": "Your application or tool initially connects to the Kafka instance using the bootstrap server and authenticates with your service account credentials. You can also access the Kafka instance by calling the REST API using a token obtained from Red Hat SSO.",
+  "kafka_service_fleet_manager_button_aria_label": "More information about the URL for Kafka Service Fleet Manager",
+  "kafka_service_fleet_manager_description": "Connect to the control plane to manage Kafka instances and service accounts, and to access metrics.",
+  "kafka_service_fleet_manager_popover_body": "Use the REST API URL to connect to the control plane to manage Kafka instances and service accounts, and to access metrics.",
+  "kafka_service_fleet_manager_popover_header": "URL for Kafka Service Fleet Manager",
+  "kafka_service_fleet_manager_url": "URL for Kafka Service Fleet Manager",
+  "more_info_about_sasl_oauthbearer": "More information about SASL/OAUTHBEARER",
+  "popover_helper_text": "For available endpoints, see the <0>API Swagger documentation</0>. For details about using this API, see the <1>API documentation</1> in the Customer Portal.",
+  "recommended": "Recommended",
+  "rest_api_description": "Streams for Apache Kafka provides two REST APIs, one for accessing the Kafka instance and one for accessing the control plane.",
```

Comment:
Control plane is not well known term outside red hat. "Managing your kafka" is more generally understandable term

Reply:
+1